.github/workflows/continues_integration.yml

```diff
@@ -21,4 +21,25 @@ jobs:
         - name: Lint
           run: npm run lint
         - name: Package
-          run: npm run vsce:package
\ No newline at end of file
+          run: npm run vsce:package
+
+  test:
+    name: Test
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+        - name: Use Node.js 10.x
+          uses: actions/setup-node@v1
+          with:
+            node-version: 10.x
+        - name: Install
+          run: npm ci --no-optional
+        - run: xvfb-run -a npm test
+          if: runner.os == 'Linux'
+        - run: npm test
+          if: runner.os != 'Linux'
```

Comment:
why?

Reply:
@dimacodota From their manual: https://code.visualstudio.com/api/working-with-extensions/continuous-integration#github-actions

Reply:
ok cool, I've missed the `xvfb-run -a npm test`